### DISCUSSION TITLE
Start Minimized to Tray?

### DISCUSSION THREAD

**Discussion by RodneyBrown** [standardnotes/forum#2762] [Suggestions]
I know you can turn on the option for Standard Notes to stay running as a System Tray app, but is there a way to have it start minimized to the tray?

I'm on Linux and I want to add Standard Note to the list of apps to autostart on login to the desktop, but I want it to start minimized to the tray.

**Comment by jasondmoss** [comment]
+1

**Comment by GagaMaga600** [comment]
+1 for Win10
I want the program to start minimized to tray when I turn on the computer. I tried different methods, from internal Windows settings, continuing with third-party programs, and ending with writing scripts and services. Nothing helps.

**Comment by tequilaj-gh** [comment]
+1
